### ISSUE TITLE
Customize syntax hilighting colors

### ISSUE THREAD

**Issue by arenol** [adobe/brackets#14289]
Can't find any card on his in Trello, so I'll post it here as a feature request.

Comming from Emacs, one of the more annoying things for me for with Brackets (which otherwise is a great source code editor), is that comments are coloured in a grey "annymous" color. 

To me, the comments are more important than the code itself. When I read code that other people has written, or code that I wrote myself long time ago, the primery thing I read is the comments. I would therefore like to at least have the possibility to customize the syntax hilighting colours.  I can't find a place to do that now.

**Comment by dayo-adebanjo** [comment]
I agree that this feature would be a pretty useful one. I've searched through the 'defaultpreferences.json' file to see if there's a way to set this up but can't seem to find one.
